backend/business-partner-agent/src/main/java/org/hyperledger/bpa/impl/IssuerManager.java

```diff
@@ -61,11 +76,23 @@
     @Inject
     BPACredentialDefinitionRepository credDefRepo;
 
+    @Inject
+    PartnerRepository partnerRepo;
+
+    @Inject
+    BPACredentialExchangeRepository credExRepo;
+
+    @Inject
+    LabelStrategy labelStrategy;
+
+    @Inject
+    Converter conv;
+
     @Inject
     RuntimeConfig config;
 
     public SchemaAPI createSchema(@NonNull String schemaName, @NonNull String schemaVersion,
-                                  @NonNull List<String> attributes, @NonNull String schemaLabel, String defaultAttributeName) {
+            @NonNull List<String> attributes, @NonNull String schemaLabel, String defaultAttributeName) {
```

Comment:
Once again just a remark. Usually I have the rule of thumb not to use more than three parameters, especially when most of them are Strings, as it makes the API not very readable. If we want to update to Java 16 we could use Records for that (which I would highly recommend), or introduce a transfer class.